CMakeModules/CPackConfig.cmake

```diff
@@ -101,5 +101,9 @@ SET(CPACK_SOURCE_IGNORE_FILES
 # Ignore build directories that may be in the source tree
 FILE(GLOB_RECURSE CACHES "${CMAKE_SOURCE_DIR}/CMakeCache.txt")
 
+SET(CPACK_WIX_LICENSE_RTF "${PROJECT_SOURCE_DIR}/LICENSE.rtf")
```

Comment:
There is no license.rtf file in ArrayFire. If you want to link to license, the it is just LICENSE (markdown).